mailinglogger/SummarisingLogger.py

```diff
@@ -108,15 +111,20 @@ def close(self):
                 FileHandler.emit(self, record)
 
         FileHandler.close(self)
-        f = os.fdopen(self.fd)
-        summary = f.read().decode(self.charset)
+        if PY3:
+            f = os.fdopen(self.fd, encoding=self.charset)
+        else:
+            f = os.fdopen(self.fd)
+        summary = f.read()
```

Comment:
I don't follow this change. `f.read()` returns bytes in both 2 and 3, why can't the original line stay?